_posts/2022-09-30-windows-macos-packaging.md

```diff
@@ -0,0 +1,32 @@
+---
+  layout: post
+  author: wetneb
+  title: "Looking for help with Windows and MacOS packaging"
+  category: Blog
+
+---
+
+OpenRefine's packaging for MacOS and Windows could be improved in many ways, and we are looking for help in this front.
+We are looking for proposals from prospective contractors to improve the install experience of OpenRefine on MacOS and/or Windows.
+After a similar effort on Ubuntu/Debian packaging, this initiative is meant to improve the user experience on other platforms, to lower the install barrier for a broader audience.
+This project is funded by an [EOSS Diversity and Inclusion grant](https://cziscience.medium.com/advancing-diversity-and-inclusion-in-scientific-open-source-eaabe6a5488b) from [CZI](https://chanzuckerberg.com/).
+
+We are hoping that those proposals could solve some of the following issues:
+* (MacOS) [Provide an Applications shortcut in the DMG distribution, with a suitable background (#5205)](https://github.com/OpenRefine/OpenRefine/issues/5205)
+* (MacOS) [Sign and notarize our DMG distribution (#2191)](https://github.com/OpenRefine/OpenRefine/issues/2191)
+* (Windows) [Offer a proper installer / uninstaller on Windows (#3224)](https://github.com/OpenRefine/OpenRefine/issues/3224)
+* (Windows) [Give an easier way to start and stop OpenRefine with a system tray integration and log viewer (#3221)](https://github.com/OpenRefine/OpenRefine/issues/3221)
+* (Windows) [Configuration for OpenRefine on Windows should use only 1 config file (.ini) (#3057)](https://github.com/OpenRefine/OpenRefine/issues/3057)
+* (Windows) [Sign openrefine.exe to eliminate extra security warnings (#3003)](https://github.com/OpenRefine/OpenRefine/issues/3003)
+
+All new packaging steps should be integrated in our Continuous Deployment infrastructure (currently running on GitHub Actions), if not in our Maven packaging configuration itself.
+
+We have funding to contract out this work to freelancers.
+To respond to this opportunity, please send the following to our developers mailing list (openrefine-dev@googlegroups.com):
```

Comment:
I'm not comfortable with this procedure. It is a large hurdle, and feels like a privacy transgression, to send a quote with price to a mailing list of 222 subscribers, many of whom we don't know at all ourselves. From the subscribers' side, I'm not sure if we should force a series of emails with quotes on them. Our goal should be to find good people, including encouraging promising but perhaps shy/vulnerable/underrepresented people to step up, and this doesn't sound like the best way to do it.

As an alternative, I suggest to ask on our user mailing list (the packaging will benefit end users) if people volunteer to be in a small short-term selection committee, and ask applicants to send applications to this group.

Reply:
I retreat my comment. Let's try it this way.

That said, I feel very responsible for having good packaging solutions for our Windows and MacOS users, and I want the right people to work on it.

If, say, by end November 2022, we haven't found suitable candidates via the above suggested procedure, then I want to step in and recruit candidates with another method.